code/game/objects/machinery/computer/camera_advanced.dm

```diff
@@ -158,6 +160,49 @@
 	eyeobj.setLoc(eyeobj.loc)
 
 
+/obj/machinery/computer/camera_advanced/proc/track(mob/living/target)
+	if(!istype(target))
+		return
+
+	tracking_target = target
+	tracking = TRUE
+
+	if(!target || !target.can_track(current_user))
+		to_chat(current_user, "<span class='warning'>Target is not near any active cameras.</span>")
+		tracking_target = null
+		return
+
+	to_chat(current_user, "<span class='notice'>Now tracking [target.get_visible_name()] on camera.</span>")
+
+	var/cameraticks = 0
+	spawn(0)
```

Comment:
Invoke async a `set waitfor = FALSE` proc.

Reply:
I'll ask ninja about this, since this is the original /tg/ camera tracking implementation that's still used.